tests/pricing_models/yieldspace_test_calc_out_given_in.py

```diff
@@ -136,12 +136,12 @@ def test_calc_out_given_in_success(self):
                     # We want to solve for the amount of bonds out given the
                     # amount of shares coming in, so we set up the problem as:
                     #
-                    #   k = (c / μ) * (μ * (z + d_z)) ** (1 - t) + (2 * y + c * z - d_y') ** (1 - t)
-                    #     = 100_100 ** (1 - T) + (300_000 - d_y') ** (1 - T)
+                    #   k = (c / mu) * (mu * (z + d_z)) ** (1 - tau) + (2 * y + c * z - d_y') ** (1 - tau)
+                    #     = 100_100 ** () + (300_000 - d_y') ** (1 - tau)
                     #
                     # Solving for d_y, we get the following calculation:
                     #
-                    #   d_y' = 300_000 - (k - 100_100 ** (1 - T)) ** (1 / (1 - T))
+                    #   d_y' = 300_000 - (k - 100_100 ** ()) ** (1 / (1 - tau))
```

Comment:
Nit: `(1 - T)` should have been updated to `(1 - tau)` but it's empty here.

Reply:
oops! good catch